### PR TITLE
5 migrate to vue

### LINKED ISSUE #5: Migrate Existing Live Game Frontend into a Vue Project.
We need to take the current `index.html` that exists in the `frontend` directory, and update it to work with a Vue project. There is a lot of content there already that exists for us to use in terms of HTML and CSS, all we would need to do is migrate it.

### PR DESCRIPTION
**Pull request by Potato-Man114**
Closes #5.

- Created Vue Project
- Using Vue components, copied the existing static HTML page.
- Created very basic routing. Live-game scoreboard is at the `#/scoreboard` route
    - For example, `localhost:5173/#/scoreboard`
- Leaving room for other tickets to be created to integrate with the scorekeepers' page and backend (or mock backend)

For code review, please look at the `frontend\src` folder. All other materials were generated by npm and vue automagically.

### CODE REVIEW

**Comment by clickclackclark** [comment]
@Potato-Man114 I was able to check your branch out locally and visit the scoreboard route. It worked great, nicely done! I personally would give my stamp of approval. This will be a solid foundation for the frontend.

**Review by jebidiah252 — APPROVED** [review]
This looks great! We can probably combine the front and back end `.gitignore` files into the same place at some point, but other than that, this looks great!

File: .gitignore
[the remark this answers is not in the document]
Comment (Potato-Man114) [reply .gitignore:1]: I've used that tool to add many more to the `frontend/.gitignore` file.

File: frontend/.vscode/settings.json
Comment (jebidiah252) [inline frontend/.vscode/settings.json:1]: I think at some point, we can start combining all of these different VS Code json files into a single workspace, but I think for now, this looks great.

**Review by jebidiah252 — APPROVED** [review]

File: frontend/.vscode/settings.json
Comment (Potato-Man114) [reply frontend/.vscode/settings.json:1]: Yes. In my ideal world we could have a devcontainer setup, but that may take more time than it is worth for this short class.

**Pull request merged**